### ISSUE TITLE
Noob question

### ISSUE THREAD

**Issue by ijsilva0123** [Mcracker/EWBF-dev-fee-to-myWallet#4]
1- In '' var net = require ("net") '', should I change '' net '' for something?
2- For Windows. I should run the server.bat. right? 
3- Should I start it, and then the original EWBF correct?
4- There's something wrong or server.bat. just gonna show the dev sheres?
INJECTION your shares: 0; dev shares/percent: 10/Infinity; total : 10
INJECTION your shares: 0; dev shares/percent: 11/Infinity; total : 11
INJECTION your shares: 0; dev shares/percent: 12/Infinity; total : 12
INJECTION your shares: 0; dev shares/percent: 13/Infinity; total : 13
INJECTION your shares: 0; dev shares/percent: 14/Infinity; total : 14
INJECTION your shares: 0; dev shares/percent: 15/Infinity; total : 15

**Comment by ijsilva0123** [comment]
This was all I changed on server.js, is that correct?
var yourPoolAddres = 's-east.equihash-hub.miningpoolhub.com./172.104.180.119';
var yourPoolPort = 20570;
var yourWalletID = 't1cAr4f5whaRswfkXeovcUoBhAVrczsJT4s'

**Comment by Trbb** [comment]
same question about 
1- In '' var net = require ("net") '', should I change '' net '' for something?

**Comment by Jaywizkid** [comment]
No, you don't need to change anything in server.js other than your wallet address, pool name/ip address, pool port.  Be sure to put all the host file entries in place from the instructions and not just one for your pool or it will not catch the DevFee shares correctly.

The only things you need to change in server.js is

Var YourPoolAddress
Var YourPoolPort
Var YourWalletID

Optionally you can change

Var YourWorkerName = '1080tiDevFee'

This will make it easy to identify the exact worker that had it's DevFee's intercepted.

**Comment by ijsilva0123** [comment]
Each 1 address of these 3 generates 7 different IP's, 3x7 = 21. You mean I have to put all 3 addresses of the Mining Pool Hub in windows '' hosts '', then the 3 addresses and the 21 IP's in '' server.js ''? So I have to capitalize all the outputs on the server, not just the devfee?

U.S.     :   us-east.equihash-hub.miningpoolhub.com :20570
Europe :   europe.equihash-hub.miningpoolhub.com :20570
Asia     :   asia.equihash-hub.miningpoolhub.com :20570

https://www.site24x7.com/public/t/results-1518545694683.html

**Comment by Jaywizkid** [comment]
I just entered the first IP for my particular pool (it has two) and that works fine. I'm not sure if you can specify more than one pool and ip in server.js, Dev will need to answer that one. Mcracker?

You only need to create one host file entry for your pool name if it is not already in the provided list in the readme.

You should make sure to copy the full host list from the readme provided and add them to your hosts file as well, as all these pools could be used to mine the DevFee shares.

**Comment by ijsilva0123** [comment]
It was a bit tricky to do that in MPH, all 3 of those addresses and 21 IP's are just for one coin, Zcash ...

**Comment by Trbb** [comment]
ok i try your way , is working now but i m not sure something
server.js show me

INJECTION your shares: 113; dev shares/percent: 0/0; total : 113
INJECTION your shares: 114; dev shares/percent: 0/0; total : 114
INJECTION your shares: 115; dev shares/percent: 0/0; total : 115
INJECTION your shares: 116; dev shares/percent: 0/0; total : 116
INJECTION your shares: 117; dev shares/percent: 0/0; total : 117
INJECTION your shares: 118; dev shares/percent: 0/0; total : 118

is normal? i guess is all share are my ,right?

by the way on ewbf.conf , i have set --fee 0

sorry my english is not good:(

**Comment by Jaywizkid** [comment]
That is correct, that means currently all your shares are yours and no DevFee shares have been intercepted yet.  Be sure you have all the host file entries provided in the readme file in place in your hosts file.  Also, no need to do --fee 0, the entire point of this is that --fee 0 is not needed because any DevFee shares are sent to your wallet anyway.

**Comment by Trbb** [comment]
Very thanks Mr. Jaywizkid . i delete --fee 0 .  and now work pretty good. :+1:

**Comment by ijsilva0123** [comment]
Dev fee 2%
INFO 01:15:57: GPU0 Accepted share 203ms [A:96, R:1]
Temp: GPU0: 60C
GPU0: 756 Sol/s
Total speed: 756 Sol/s
INFO 01:16:15: GPU0 Accepted share 203ms [A:97, R:1]
INFO 01:16:38: GPU0 Accepted share 203ms [A:98, R:1]
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
INJECTION your shares: 97; dev shares/percent: 2/2.06; total : 99
INJECTION your shares: 98; dev shares/percent: 2/2.04; total : 100
